_posts/2019-06-05-hibernate-search-extension.adoc

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: 'New extension: Hibernate Search + Elasticsearch'
+date: 2019-06-05
+tags: quarkus hibernate elasticsearch
+author: gsmet
+---
+
+In the 0.15.0 version of Quarkus, we introduced the Hibernate Search + Elasticsearch extension and, just a few days ago, we added link:/guides/hibernate-search-guide[a guide] and https://github.com/quarkusio/quarkus-quickstarts/tree/master/hibernate-search-elasticsearch[a quickstart] explaining how to use it.
+
+Note that this extension is still experimental as it is based on an Alpha version of Hibernate Search 6.
+While the code is stable and thoroughly tested, some features are still missing and we might change some APIs along the way to the final version - even if the Hibernate team has made a great effort to stabilize them before the inclusion in Quarkus.
+
+If you don't mind updating your code from time to time, you're very welcome to become an early adopter and provide feedback.
```

Comment:
What are the benefits of Hibernate Search 6 ? 
In short better marketing why people should jump on this ...

This line mentions drawback of updating your code from time to time and expectation to provide feedback.

Reply:
Well, it's basically Search redesigned to support both Lucene and Elasticsearch properly. Right now, apart from the fact that it's very different, you won't see a lot of advantages. They will come later.

Reply:
If you don't mind updating your code from time to time, you're very welcome to become an early adopter and provide feedback. 
  =>
If you want proper support for both Lucene and Elasticsearch and don't mind updating your code from time to time, you're very welcome to become an early adopter and provide feedback.

wdyt ?

Reply:
ah, finished reading the blog post, the above proposed change doesn't make sense, new proposal:

If you want proper support for Elasticsearch on Quarkus and don't mind updating your code from time to time, you're very welcome to become an early adopter and provide feedback.